packages/breadcrumb/breadcrumb.js

```diff
@@ -0,0 +1 @@
+export default function Breadcrumbs () {}
```

Comment:
Decide, breadcrumbS or breadcrumb

Reply:
Fixed in 2c471a0b8a946dedd9dc764e0b407586e592ccbf.